governance/cla/index.html

```diff
@@ -1,3 +1,202 @@
----
-redirect_to: "/governance/cla/cla.pdf"
----
+<!DOCTYPE html>
+<html lang="en-gb">
+    <head>
+        <title>Fiduciary License Agreement 2.0 - Libero Governance</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="apple-touch-icon" sizes="57x57" href="assets/favicons/apple-touch-icon-57x57.png">
+        <link rel="apple-touch-icon" sizes="60x60" href="assets/favicons/apple-touch-icon-60x60.png">
+        <link rel="apple-touch-icon" sizes="72x72" href="assets/favicons/apple-touch-icon-72x72.png">
+        <link rel="apple-touch-icon" sizes="76x76" href="assets/favicons/apple-touch-icon-76x76.png">
+        <link rel="apple-touch-icon" sizes="114x114" href="assets/favicons/apple-touch-icon-114x114.png">
+        <link rel="apple-touch-icon" sizes="120x120" href="assets/favicons/apple-touch-icon-120x120.png">
+        <link rel="apple-touch-icon" sizes="144x144" href="assets/favicons/apple-touch-icon-144x144.png">
+        <link rel="apple-touch-icon" sizes="152x152" href="assets/favicons/apple-touch-icon-152x152.png">
+        <link rel="apple-touch-icon" sizes="180x180" href="assets/favicons/apple-touch-icon-180x180.png">
+        <link rel="icon" type="image/svg+xml" href="assets/favicons/favicon.svg">
+        <link rel="icon" type="image/png" sizes="32x32" href="assets/favicons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="192x192" href="assets/favicons/android-chrome-192x192.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="assets/favicons/favicon-16x16.png">
+        <link rel="shortcut icon" href="assets/favicons/favicon.ico">
+        <link rel="manifest" href="assets/favicons/manifest.json">
+
+        <link rel="stylesheet" type="text/css" media="screen" href="../assets/css/all.css" />
+        <link rel="stylesheet" type="text/css" media="print" href="../assets/css/print.css">
+    </head>
+    <body>
+
+    <div class="global-wrapper">
+      <main class="main">
+
+        <div class="wrapper wrapper--content">
+          <div class="grid">
+            <div class="content-container grid__item one-whole large--ten-twelfths push--large--one-twelfth x-large--eight-twelfths push--x-large--two-twelfths grid-column">
+              <section class="text-center">
+                <h2 id="tmp-title">Fiduciary License Agreement 2.0</h2>
```

Comment:
Lines 36-39 should be a single `<h1>`, and all subsequent headings should be promoted one level.